src/MSONOneOfParser.cc

```diff
@@ -12,121 +12,119 @@
 
 namespace snowcrash {
 
-    /** Implementation of processNestedSection */
-    MarkdownNodeIterator SectionProcessor<mson::OneOf>::processNestedSection(const MarkdownNodeIterator& node,
-                                                                             const MarkdownNodes& siblings,
-                                                                             SectionParserData& pd,
-                                                                             const ParseResultRef<mson::OneOf>& out) {
+/** Implementation of processNestedSection */
+MarkdownNodeIterator SectionProcessor<mson::OneOf>::processNestedSection(
```

Comment:
There are parameters breaked on "indentation" while in many other cases is break on "parenthesis"
```
MarkdownNodeIterator
SectionProcessor<mson::OneOf>::processNestedSection(const MarkdownNodeIterator &node, 
                                                                                             const MarkdownNodes &siblings,
```